static/js/util/factories/websites.ts

```diff
@@ -38,25 +41,60 @@ export const makeWebsiteConfigField = (
   }
 }
 
-export const makeWebsiteConfigItem = (name: string): ConfigItem => ({
-  fields: [
-    {
-      label:  "Title",
-      name:   "title",
-      widget: WidgetVariant.String
-    },
-    {
-      label:  "Content",
-      name:   "content",
-      widget: WidgetVariant.Markdown
-    }
-  ],
+const exampleFields = [
+  {
+    label:  "Title",
+    name:   "title",
+    widget: WidgetVariant.String
+  },
+  {
+    label:  "Description",
+    name:   "description",
+    widget: WidgetVariant.Text
+  },
+  {
+    label:  "Content",
+    name:   "markdown",
+    widget: WidgetVariant.Markdown
+  }
+]
+
+export const makeWebsiteConfigItem = (name?: string): ConfigItem => ({
+  fields:   cloneDeep(exampleFields),
   folder:   casual.word,
   label:    casual.word,
   name:     name || casual.word,
   category: casual.word
 })
 
+export const makeFileConfigItem = (name?: string): SingletonConfigItem => ({
+  fields: cloneDeep(exampleFields),
+  file:   casual.word,
+  label:  casual.word,
+  name:   name || casual.word
+})
+
+export const makeTopLevelConfigItem = (name?: string): TopLevelConfigItem => ({
+  fields:   cloneDeep(exampleFields),
+  name:     name || casual.word,
+  label:    casual.word,
+  category: casual.word,
+  ...(Math.random() >= 0.5 ?
```

Comment:
Can we add a parameter which says which kind of config item this is, and use `Math.random` if the parameter is not set?  It would be useful to be help avoid flaky tests